app/models/site.rb

```diff
@@ -24,4 +28,11 @@ def instance
       first_or_create
     end
   end
+
+  private
+
+    def update_banner_config
+      # If a site banner_image uploaded, then update Sufia's banner_image to use it
+      Sufia.config.banner_image = banner_image? ? banner_image.url : nil
```

Comment:
I'm not sure what `Sufia.config` is. Does that get persisted somewhere, or do we need to do this every request?